src/proof/linked_list.rs

```diff
@@ -0,0 +1,248 @@
+//! Double LinkedList.
```

Comment:
we are going to remove this since we will try to use https://docs.rs/mrklt first.